src/validate.ts

```diff
@@ -0,0 +1,67 @@
+import { check, CheckResult } from '@superfaceai/cli/dist/logic/check';
+
+import { allProfileProviderCombinations, loadSuperJson } from './util';
+
+export async function checkCapabilities(): Promise<CheckResult[]> {
+  const checkCombinations = allProfileProviderCombinations();
+  const superJson = loadSuperJson();
+
+  const results: CheckResult[] = [];
+
+  const map: {
+    variant?: string; // no reason for variants in Station
+  } = {};
+
+  for (const checkCombination of checkCombinations) {
+    try {
+      const checkResults = await check(
+        superJson,
+        checkCombination.profile,
+        checkCombination.provider,
+        map
+      );
+
+      if (checkResults.length > 0) {
+        checkResults.forEach(result => {
```

Comment:
Could we use `results.push(...checkResults)` here? 🤷

Reply:
Oh yes, I am too oldschool.

Reply:
OFC, I am just too oldschool.

Reply:
And I should stop using vscode extension, because it hiccups too often.